com.tic80.TIC_80.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: com.tic80.TIC_80
+runtime: org.freedesktop.Platform
+sdk: org.freedesktop.Sdk
+runtime-version: '21.08'
+command: tic80
+rename-desktop-file: tic80.desktop
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --device=dri
+modules:
+  - name: TIC-80
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DBUILD_PRO=On
```

Comment:
I don't think this is fair, at least until payments support on Flathub is available? The author sells the pro version for $10 on Itch and I believe it should be disabled in the Flathub build for now.

What's your opinion about it @nesbox?

Reply:
I think PRO should be disabled by default because we use PRO payments to support TIC-80.

Reply:
ok, I removed the pro version.